x-pack/plugin/core/src/main/java/org/elasticsearch/license/XPackLicenseState.java

```diff
@@ -629,12 +668,49 @@ public boolean isAllowedByLicense(OperationMode minimumMode, boolean needActive)
 
     /**
      * A convenient method to test whether a feature is by license status.
-     * @see #isAllowedByLicense(OperationMode, boolean)
      *
-     * @param minimumMode  The minimum license to meet or exceed
+     * @param minimumMode The minimum license to meet or exceed
+     * @see #isAllowedByLicense(OperationMode, boolean)
      */
     public boolean isAllowedByLicense(OperationMode minimumMode) {
         return isAllowedByLicense(minimumMode, true);
     }
 
+    public static class FeatureUsage {
```

Comment:
This class has no explicit info about whether the feature is "on-by-default". Does it matter? Or it's more of a documentation issue?